packages/saber-plugin-feed/lib/index.js

```diff
@@ -84,11 +84,19 @@ exports.apply = (api, options = {}) => {
 
     const feedLinks = {}
 
+    if (jsonFeedPath) {
+      feedLinks.json = resolveURL(siteConfig.url, jsonFeedPath)
+    }
+
+    if (atomFeedPath) {
+      feedLinks.atom = resolveURL(siteConfig.url, atomFeedPath)
+    }
+
     // Feed instance
     const feed = new Feed({
       title: siteConfig.title,
       description: siteConfig.description,
-      id: siteConfig.url,
+      id: siteConfig.url.replace(/\/?$/, '/'), // Ensure that the id ends with a slash
```

Comment:
why?

Reply:
ID's must be in canonical form https://validator.w3.org/feed/docs/warning/NonCanonicalURI.html
It complained when it didn't end with a /